seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -80,7 +80,8 @@
         "jQuery often selects an HTML element with a <code>selector</code>, then does something to that element.",
         "For example, let's make all of your <code>button</code> elements bounce. Just add this code inside your document ready function:",
         "<code>$(\"button\").addClass(\"animated bounce\");</code>",
-        "Note that we've already included both the jQuery library and the Animate.css library in the background so that you can use them in the editor. So you are using jQuery to apply the Animate.css <code>bounce</code> class to your <code>button</code> elements."
+        "Note that we've already included both the jQuery library and the Animate.css library in the background so that you can use them in the editor. So you are using jQuery to apply the Animate.css <code>bounce</code> class to your <code>button</code> elements.",
+        "Additionally make sure to use <code>$(\"button\").addClass(\"animated bounce\");</code> instead of <code>$(&#39;button&#39;).addClass(\"animated bounce\");</code> since single-quote selectors will not pass our tests."
```

Comment:
I'm not sure why this went up to 23 commits. All I did was follow the directions to squash. Would the best option just be to delete my repo and try again?